src/LibraryManager/Providers/Cdnjs/CdnjsProvider.cs

```diff
@@ -235,17 +235,17 @@ public ILibraryCatalog GetCatalog()
             return null;
         }
 
-        private string GetLibraryName(ILibraryInstallationState state)
+        public string GetLibraryName(string libraryId)
```

Comment:
>GetLibraryName [](start = 22, length = 14)

Would it make sense to instead have one method like 

public void GetLibraryNameAndVersion(string libraryName, out string name, out string version)

Reply:
Yes, this is another bug. I started implementing this as part of #123 But, this won't make to Escrow bar so leaving the proper fix to preview4